### PR TITLE
spectrwm: update to 3.6.0.

### PR DESCRIPTION
**Pull request by Rooyca**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/spectrwm/template [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
-conf_files="/etc/spectrwm.conf"
+checksum=49854c5215475d2450d2558da12a3afe9019861170ba9d556c8b403d4d66f0d1
 
 if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	makedepends+=" musl-legacy-compat"
+else
+	makedepends+=" libbsd-devel"
 fi
 
+do_install() {
```
Comment (classabbyamp) [inline srcpkgs/spectrwm/template:25]: don't override do_install, use post_install instead

File: srcpkgs/spectrwm/template [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
 makedepends="libXcursor-devel libXft-devel libXrandr-devel libXt-devel
- xcb-util-keysyms-devel xcb-util-wm-devel"
+ xcb-util-keysyms-devel xcb-util-wm-devel libbsd-devel"
 short_desc="Small dynamic tiling window manager for X11"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="ISC"
 homepage="http://spectrwm.org"
 distfiles="https://github.com/conformal/spectrwm/archive/${_distname}.tar.gz"
-checksum=7fe4260208a52d9fb844291d0afe6af18887c5452df0b921ed780af7b9223f0f
-conf_files="/etc/spectrwm.conf"
```
Comment (classabbyamp) [inline srcpkgs/spectrwm/template:18]: why remove this?

**Comment by classabbyamp** [comment]
> I tested the changes in this PR: YES

how did you test this? the current template removes the spectrwm binary

File: srcpkgs/spectrwm/template (hunk 2 above)
Comment (Rooyca) [reply srcpkgs/spectrwm/template:18]: I removed it for some reason and forgot to add it back.

File: srcpkgs/spectrwm/template [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
 homepage="http://spectrwm.org"
 distfiles="https://github.com/conformal/spectrwm/archive/${_distname}.tar.gz"
-checksum=7fe4260208a52d9fb844291d0afe6af18887c5452df0b921ed780af7b9223f0f
+checksum=49854c5215475d2450d2558da12a3afe9019861170ba9d556c8b403d4d66f0d1
 conf_files="/etc/spectrwm.conf"
 
 if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	makedepends+=" musl-legacy-compat"
+else
+	makedepends+=" libbsd-devel"
```
Comment (classabbyamp) [inline srcpkgs/spectrwm/template:23]:
```suggestion
```

redundant to the change on line 11

File: srcpkgs/spectrwm/template [hunk 4]
```diff
@@ -1,26 +1,32 @@
...
 fi
 
 post_install() {
 	vlicense LICENSE.md
+	vdoc spectrwm.html
+	vinstall spectrwm.desktop 644 usr/share/xsessions
+	vinstall baraction.sh 755 usr/share/spectrwm
+	vinstall screenshot.sh 755 usr/share/spectrwm
```
Comment (classabbyamp) [inline srcpkgs/spectrwm/template:31]:
```suggestion
```
spectrwm.html is just an html version of the manpage already in the package.

the makefile already installs the other files. this is redundant

**Pull request merged**